### ISSUE TITLE
"Show TeX" is a disaster for large expressions

### ISSUE THREAD

**Issue by elmisback** [herbie-fp/odyssey#106]
Example expression: `sqrt(x+1) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x) - sqrt(x)`
Show TeX:
![image](https://github.com/user-attachments/assets/f30a485d-9f4b-4145-962f-a47bf365d7b7)

**Comment by jaelafield** [comment]
Don't let the element become super large. Add scroll or other restriction to box size to prevent super long expression from manipulating page spacing.

Fix after #158

**Comment by jaelafield** [comment]
Adding note that, if this fix doesn't address it, should make issue for big expressions causing the error plot half of screen to be really small. I looked around and didn't see another issue for this, though I may have missed one. 

Options to fix: make center divide draggable (may be a valuable feature regardless), so plot/expression table portions can be expanded/shrunk, or put limit on size of expressions table

Otherwise, haven't made any progress yet.

**Comment by elmisback** [comment]
You can go ahead and make that issue, we will close it if it gets fixed!

**Comment by jaelafield** [comment]
FPBench server seems to do the same unnecessary-()-ing issue, most noticeably ugly for long expressions

**Comment by elmisback** [comment]
I think I fixed this
